pygsti/modelmembers/povms/__init__.py

```diff
@@ -418,7 +422,7 @@ def povm_type_from_op_type(op_type):
     return povm_type_preferences
 
 
-def convert(povm, to_type, basis, ideal_povm=None, flatten_structure=False):
+def convert(povm, to_type, basis, cp_penalty=1e-7, ideal_povm=None, flatten_structure=False):
```

Comment:
When adding keyword arguments it's good to append them to the existing argument sequence. This way if someone uses positional arguments they don't get an error when using updated code. In this situation, placing the new cp_penalty keyword argument before the existing ideal_povm keyword argument could lead to errors for people specifying ideal_povm by its position.